src/baseline_bots/utils.py

```diff
@@ -414,8 +414,13 @@ def get_best_orders(bot, proposal_order: dict, shared_order: dict):
     
     def __deepcopy__(game):
```

Comment:
It would be nice to see a test for this function

Reply:
this function is directly copied from the main diplomacy. I did not do much of any update  just checking stance